lib/checks.js

```diff
@@ -33,9 +33,30 @@ async function create(octokit, data) {
 
   options.check_run_id = createResult.data.id;
 
-  // artificially wait for 15 seconds
-  await new Promise((resolve) => setTimeout(resolve, 15000));
+  const commit = await octokit.rest.git.getCommit({owner:data.owner, repo:data.repo, commit_sha: data.head_sha});
+  const commitMessage = commit.data.message;
 
+  const hasWpt = /https:\/\/wpt\.fyi/.test(commitMessage);
```

Comment:
Additional ways that people might link to WPT:
- http://wpt.live/css/cssom/historical.html
- https://github.com/web-platform-tests/wpt/blob/master/css/cssom/historical.html (also branch name will eventually change from `master` to `main`)
- Links to PRs like https://github.com/web-platform-tests/wpt/pull/11163

No doubt we will discover more. Can you add a TODO block listing these?